src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/non_max_suppression.hpp

```diff
@@ -27,21 +27,32 @@ using NmsParams = std::tuple<InputShapeParams,  // Params using to create 1st an
                              float,             // Score threshold
                              float,             // Soft NMS sigma
                              ov::op::v5::NonMaxSuppression::BoxEncodingType,  // Box encoding
-                             bool,                                                // Sort result descending
+                             bool,                                            // Sort result descending
                              ov::element::Type,                               // Output type
-                             std::string>;                                        // Device name
+                             std::string>;                                    // Device name
 
 class NmsLayerTest : public testing::WithParamInterface<NmsParams>, virtual public ov::test::SubgraphBaseStaticTest {
 public:
     static std::string getTestCaseName(const testing::TestParamInfo<NmsParams>& obj);
 protected:
     void SetUp() override;
+    void compare(const std::vector<ov::Tensor>& expected, const std::vector<ov::Tensor>& actual) override;
+
+    size_t num_batches;
+    size_t num_boxes;
```

Comment:
use `m_` naming style to differentiate with local variables. Or don't use variables at all, such information may be got via `GetParam()` called inside `compare()`

Reply:
done